datajunction-server/datajunction_server/sql/decompose.py

```diff
@@ -154,7 +183,7 @@ def update_ast(func, measures: list[Measure]):
                         args=[ast.Column(ast.Name(measures[0].name))],
                     ),
                     right=ast.Function(
-                        ast.Name("COUNT"),
+                        ast.Name("SUM"),
```

Comment:
I think `COUNT` was correct... oh wait, no this is to aggregate the components of the expression, right? And to aggregate count's we would still use a sum.

Reply:
Yeah, I think in this case we actually have to use `SUM`, because we need to sum together all the counts for it to be correct.